dev-infrastructure/swift/sal_env_vars

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+error_msg() {
+    echo "Error: $1"
+    exit 1
+}
+
+is_dev_subscription() {
+    return $(test "$(az account show --query name --output tsv)" = "ARO Hosted Control Planes (EA Subscription 1)")
+}
+
+is_redhat_user(){
+    return $(test "$(az account show --query user.name --output tsv)" = "jaharrin@redhat.com")
```

Comment:
Did you mean to check your own full redhat user here or should this check for *@redhat.com?

Reply:
I'll fix that, thanks!